Gemfile.lock

```diff
@@ -118,8 +107,7 @@ GEM
     yell (2.2.2)
 
 PLATFORMS
-  ruby
-  x86_64-linux
+  arm64-darwin-23
```

Comment:
I don't think we want this `PLATFORMS` change. TBH I'm not sure what the best practice is for supporting multiple platforms: maybe we _add_ arm64-darwin-23 but don't replace the others? I'm looking at https://blog.saeloun.com/2022/08/16/understanding_gemfile_and_gemfile_lock/#platforms-1